thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1291,6 +1291,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                 activity.inactiveStats.sumStatistic(StNumIndexSeeks, keyManager->querySeeks()-startSeeks);
                 activity.inactiveStats.sumStatistic(StNumIndexScans, keyManager->queryScans()-startScans);
                 activity.inactiveStats.sumStatistic(StNumIndexWildSeeks, keyManager->queryWildSeeks()-startWildSeeks);
+                activity.contextLogger.updateStatsDeltaTo(activity.inactiveStats);
```

Comment:
This isn't going to work in this context because it's relying on the state of 'inactiveStats' for the previous values to calculate the delta.
However, inactiveStats is being updated from multiple threads, so when it gets here via the scoped dtor,
it will not see the delta between when started and current, but whatever the inactiveStats values are and now.

It will get away with it in indexread and indexdistrib, because it's single threaded.

Similar to the way the existing code records start values and uses them to delta this wiill need to do same.
It can still use CRuntimeStatisticCollection::updateDelta, but must base in a record 'previous' stats collection.
e.g. you could replace the current lines that are preserving start values, with:

CRuntimeStatisticCollection startStats(contextLogger.queryStats());